cluster-agent/values.yaml

```diff
@@ -34,11 +40,8 @@ agent:
   snapshotSyncInterval: 15
   agentServerPort: 8989
 
-
-
 serviceAccount:
   create: true
   name: appdynamics-operator
 
-metrics-server:
-  install: true
```

Comment:
Removed as these configure values are not used in the chart.
Removing them because these unused configure parameters will appear in ICP UI configure page. They will confuse users who set them and see no effect in the chart.